Book/php7/internal_types/zvals/basic_structure.rst

```diff
@@ -194,7 +194,7 @@ we'll get back to it later.
 
 Knowing the zval structure you can now write code making use of it::
 
-    zval zv_ptr = /* ... get zval from somewhere */;
+    zval *zv_ptr = /* ... get zval from somewhere */;
 
     if (zv_ptr->type == IS_LONG) {
```

Comment:
This should reference `zv_ptr->u1.v.type`, `zv_ptr->type` doesn't exist...

Reply:
Oh, I missed that line. Just fixed it.